roaring/src/bitmap/store/mod.rs

```diff
@@ -25,13 +25,18 @@ pub enum Store {
     Bitmap(BitmapStore),
 }
 
-pub enum Iter<'a> {
+enum IterInner<'a> {
     Array(slice::Iter<'a, u16>),
     Vec(vec::IntoIter<u16>),
     BitmapBorrowed(BitmapIter<&'a [u64; BITMAP_LENGTH]>),
     BitmapOwned(BitmapIter<Box<[u64; BITMAP_LENGTH]>>),
 }
 
+pub struct Iter<'a> {
+    inner: IterInner<'a>,
+    size_hint: u64,
```

Comment:
Sorry you went to all the trouble of adding tracking the size hint, but with #295, this can all go back away now 😢